.kitchen.yml

```diff
@@ -16,9 +16,33 @@ platforms:
   - name: ubuntu-18.04
   - name: centos-6
   - name: centos-7
+  - name: windows-2012r2
```

Comment:
How should the windows platform be approached for testing? They were not added to the `.travis.yml` file, but have been tested locally.

Reply:
Great question. Let me take a look at the Travis setup and see what makes sense.

Reply:
Since we'll be using appveyor for windows tests, no need to worry about travis in this case